server/backend/api/app/crud/crud_utils.py

```diff
@@ -100,3 +101,31 @@ def is_app_admin(
     if access_roles and admin_role in access_roles:
         return True
     return False
+
+
+def get_aws_target_env() -> AwsTargetEnv:
+    # target_env is assigned from gov's AWS platform, does not exist in local (None).
+    return os.environ.get("target_env")
+
+
+def is_on_aws_prod() -> bool:
+    return get_aws_target_env() == AwsTargetEnv.PROD
+
+
+def use_api_instance_by_app(application: models.FamApplication) -> ApiInstanceEnv:
+    """
+    FAM PROD environment supports (DEV/TET/PROD) integrated applications.
+    Only PROD application at FAM PROD uses API instance in PROD.
+    Lower FAM environment uses only TEST instance.
+    Ref @FAM Wiki: https://github.com/bcgov/nr-forests-access-management/wiki/Environment-Management
+    """
+    api_instance_env = ApiInstanceEnv.TEST  # API TEST instance as default.
+    if (is_on_aws_prod() and (
+        # either PROD app or app is FAM
+        application.app_environment == AppEnv.APP_ENV_TYPE_PROD or
+        application.application_name == APPLICATION_FAM
```

Comment:
so I pass application instead of the app_env, so we can check by application_name